cephci/cluster_info.py

```diff
@@ -167,40 +166,17 @@ def get_ceph_var_logs(cluster, log_dir):
     ceph cluster var logs into log directory.
     """
     for node in cluster.get_nodes():
-        download_dir = os.path.join(log_dir, "ceph_logs", node.hostname)
+        download_dir = os.path.join(log_dir, "ceph_logs")
```

Comment:
`os.path.join` and `os.makedirs` are same across the loop. Move these steps out of loop